operator/src/main/java/oracle/kubernetes/operator/helpers/JobHelper.java

```diff
@@ -117,19 +119,40 @@ Domain getDomain() {
    * @param next Next processing step
    * @return Step for creating job
    */
+  @Deprecated
```

Comment:
Does anyone call this method?  Let's just delete it.

Reply:
will do.